### ISSUE TITLE
what does `kanban show status` do?

### ISSUE THREAD

**Issue by jowodo** [coderofsalvation/kanban.bash#38]
Hi! 

thanks for creating and maintaining this awesome piece of software!
What does `kanban show status` do? Or what is it supposed to do? This is my experience: 
``` 
[pur@lithium ~]$ kanban show 

┌──────┐              ┌──────┐              ┌───────┐             ┌──────┐              ┌───────┐             ┌─────────┐
│ TODO │_______       │ HOLD │_______       │ DOING │_______      │ DONE │_______       │ NOTES │_______      │ BACKLOG │_____
│                     │                     │                     │                     │                     │
│ 2   #linu encr                                                  │ 1   #test 1                               
│ 3   #test test                                                                                              

[pur@lithium ~]$ kanban show status 
/home/pur/.local/bin/kanban: line 256: cd: status: No such file or directory
[kanban] showing nested kanban 'status'

┌──────┐              ┌──────┐              ┌───────┐             ┌──────┐              ┌───────┐             ┌─────────┐
│ TODO │_______       │ HOLD │_______       │ DOING │_______      │ DONE │_______       │ NOTES │_______      │ BACKLOG │_____
│                     │                     │                     │                     │                     │
│ 2   #linu encr                                                  │ 1   #test 1                               
│ 3   #test test                                                                                              

[pur@lithium ~]$ 
``` 

Could this maybe be some code relating to old config? 
Thanks!

**Comment by coderofsalvation** [comment]
Hi I'ved pushed a fix which makes this feature [finally] shine :D 
The idea is that you can organize/access multiple kanbans easily, via symbolic links and/or subfolders.
I've updated the description/command to the following workflow:

```
Usage:

  kanban init                             # initialize kanban in current directory
  kanban add                              # add item interactive (adviced) 
  kanban show [dir with .kanban]          # show global [or nested] ascii kanbanboard
  ...
``` 

So let's say you have multiple projects in a `/project` folder:

```
$ cd ~/project
$ kanban init
$ mkdir projectA && cd projectA && kanban init && cd -
$ mkdir projectB && cd projectB && kanban init 
$ mkdir projectC && cd projectC && kanban init && cd ~/project
$ kanban show

| TODO |       | HOLD |
....

kanbans: 
   ├─ projectA
   ├─ projectB/projectC
```

So now you can run `kanban show projectA` or `kanban show projectB` or `kanban show projectB/projectC` to list/manage those kanban boards from here.

> TLDR: it saves you from hopping between different kanban-directories to run the 'kanban' command

**Comment by coderofsalvation** [comment]
also read more here: https://github.com/coderofsalvation/kanban.bash?tab=readme-ov-file#nested-kanbans
